Assets/Scripts/SaveSystem/Dto/GameData.cs

```diff
@@ -11,7 +11,14 @@ public class GameData
 
         public override string ToString()
         {
-            var players = string.Join(", ", PlayersData.Select(playerData => playerData.ToString()));
+            var players = string.Join(", ", PlayersData
+                .Select(playerData => playerData
+                .ToString()));
+
+            var enemies = string.Join(", ", EnemiesData
+                .Select(enemiesData => enemiesData
+                .ToString()));
+
             return players;
```

Comment:
You collected a line of enemies, but at the same time you return only players

Reply:
+?